components/prize_pool/commons/prize_pool.lua

```diff
@@ -955,10 +955,10 @@ function Placement:init(args, parent, lastPlacement)
 	-- Use the last known place and set the place range based on the entered number of opponents
 	if not self.placeStart and not self.placeEnd then
 		self.placeStart = lastPlacement + 1
-		self.placeEnd = lastPlacement + #self.opponents
+		self.placeEnd = lastPlacement + math.max(#self.opponents, 1)
 	end
 
-	assert(#self.opponents > self.placeEnd - self.placeStart, 'Placement: Too many opponents')
+	assert(#self.opponents <= 1 + self.placeEnd - self.placeStart, 'Placement: Too many opponents')
```

Comment:
```suggestion
	assert(#self.opponents < self.placeEnd - self.placeStart, 'Placement: Too many opponents')
```

Reply:
if you want `<` instead of `<=` you need `+2` not `+0`

3 <= 2+1 is correct
3 < 2 is not

Reply:
why the + 1 is needed with `<=`

since both start and end are inclusive you cut off 1 to many when subtracting them to get the range

examples:
place=1
—> start=1; end=1
—> delta=0, but the range is 1
place=3-4
—> start=3; end=4
—> delta=1, but the range is 2